repos/system_upgrade/el7toel8/actors/checkbrltty/actor.py

```diff
@@ -0,0 +1,48 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.libraries.common.reporting import report_with_remediation, report_generic
+from leapp.models import InstalledRedHatSignedRPM, BrlttyMigrationDecision
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckBrltty(Actor):
+    """
+    Check if brltty is installed, check whether configuration update is needed.
+    """
+
+    name = 'check_brltty'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report, BrlttyMigrationDecision,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        for fact in self.consume(InstalledRedHatSignedRPM):
+            for rpm in fact.items:
+                if rpm.name == 'brltty':
```

Comment:
Now that has_package function has been introduced this can be simplified a bit. 
You can check for brltty package with 
```
from leapp.libraries.common.rpms import has_package
...
def process(self):
    if has_package(InstalledRedHatSignedRPM, 'brltty'):
       # do something
```
Like in https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/actors/checkgrep/actor.py